Gemfile

```diff
@@ -18,3 +18,5 @@ gem 'overcommit', '0.58.0'
 gem 'rubocop', '0.79.0'
 
 gem 'coveralls', require: false
+
+gem 'pry'
```

Comment:
Any hard reason we need this? `binding.irb` gets you most of the functionality without an additional dev dependency.

Reply:
Honestly, I've always used pry so I didn't know about `binding.irb`. I wasn't planning on leaving it there in the final commit anyways, but I'll take a look at `binding.irb`